backend/lambda/function.mjs

```diff
@@ -39,66 +49,114 @@ export const handler = async (event, context) => {
     try {
         switch (event.requestContext.routeKey) {
             case "POST /levels":
-                // TODO Actual Implementation
                 requestJSON = JSON.parse(event.body);
-                var levelName = requestJSON.name; 
-                body = {
+                var levelName = requestJSON.name;
+                if (!levelName) throw new BadRequestException(`'name' must be provided in the request.`);
+                var levelCreator = requestJSON.creator;
```

Comment:
Question about key names:

Should we unify key names? Eg. `levelName` -> `name` and `levelCreator` -> `creator`? Then we dont need to transform them all the time?

Reply:
We could, if you think that's cleaner. Either way, I think I like having a variable set by pulling out the value from the JSON - so I don't think that'd save on transforming (but not necessary).

 And there are a couple ("name", for instance) where there's a Level Name vs a Creator Name.